helpers/validators/articleValidators.js

```diff
@@ -11,7 +12,8 @@ const articleSchema = {
     .required(),
   body: Joi.string()
     .trim()
+    // .min(100)
```

Comment:
remove the comment if this validation is not necessary.

Reply:
Okay, sorry for I didn't remember to remove the comments, I'm gonna do that.